.github/fabricbot.json

```diff
@@ -4101,6 +4101,77 @@
                 }
               }
             ]
+          },
+          {
+            "operator": "and",
+            "operands": [
+              {
+                "operator": "not",
+                "operands": [
+                  {
+                    "name": "hasLabel",
+                    "parameters": {
+                      "label": "os-android"
+                    }
+                  }
+                ]
+              },
+              {
+                "operator": "not",
+                "operands": [
+                  {
+                    "name": "hasLabel",
+                    "parameters": {
+                      "label": "os-maccatalyst"
+                    }
+                  }
+                ]
+              },
+              {
+                "operator": "not",
+                "operands": [
+                  {
+                    "name": "hasLabel",
+                    "parameters": {
+                      "label": "os-ios"
+                    }
+                  }
+                ]
+              },
+              {
+                "operator": "not",
+                "operands": [
+                  {
+                    "name": "hasLabel",
+                    "parameters": {
+                      "label": "os-tizen"
+                    }
+                  }
+                ]
+              },
+              {
+                "operator": "not",
+                "operands": [
+                  {
+                    "name": "hasLabel",
+                    "parameters": {
+                      "label": "os-tvos"
+                    }
+                  }
+                ]
+              },
+              {
+                "operator": "not",
+                "operands": [
+                  {
+                    "name": "hasLabel",
+                    "parameters": {
+                      "label": "arch-wasm"
```

Comment:
Is there a way to allow `arch-wasm` if the area label is `area-System.Runtime.Intrinsics`.

That is one of the cases where we have overlap in ownership responsibilities and where I do need to be involved on many of them.

Reply:
Eww, not cleanly. I could remove the wasm exclusion rule across all your areas, not it would be hard to achieve for a single area.

Reply:
I'm fine with WASM being tagged across all my areas. I can always keep filtering them out manually.

Just don't want to be in a scenario where I miss API tweaks or changes to an area we're actively working on.

Reply:
I got this handled with dotnet/fabricbot-config#78